nncf/quantization/algorithms/weight_compression/activation_stats.py

```diff
@@ -8,35 +8,40 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+from dataclasses import dataclass
 from typing import List, Tuple
 
 from nncf.tensor import Tensor
 from nncf.tensor import functions as fns
 
 
-def process_stats(stats: List[Tensor], subset_size: int) -> Tuple[Tensor, Tensor]:
+@dataclass
+class WCStatistics:
```

Comment:
Could you align this class with  ?https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/common/tensor_statistics/statistics.py#L22

Reply:
Done